Sources/Document/Change/Checkpoint.swift

```diff
@@ -28,17 +28,17 @@ struct Checkpoint: Equatable {
      */
     static let initial = Checkpoint(serverSeq: 0, clientSeq: 0)
     private var serverSeq: Int64
-    private var clientSeq: Int
+    private var clientSeq: UInt32
 
-    init(serverSeq: Int64, clientSeq: Int) {
+    init(serverSeq: Int64, clientSeq: UInt32) {
```

Comment:
Is serverSeq's type different from clientSeq' type?

Reply:
Yes. see the protobuf message below.

https://github.com/yorkie-team/yorkie-ios-sdk/blob/7084ee76600d7939ea99b867ebe663e536a7eb5c/Sources/API/V1/Protos/resources.proto#L275-L278